.eslintrc

```diff
@@ -14,4 +11,10 @@
       "ObjectExpression": "first"
     }]
   },
+  "parserOptions": {
+    "ecmaVersion": 2018
```

Comment:
This means that ESLint will no longer complain if you use JS features added after ES2015.